src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -24,9 +23,9 @@ class FileLinkFormatter implements \Serializable
     private $fileLinkFormat;
     private $requestStack;
     private $baseDir;
-    private $urlFormat;
+    private $queryString;
 
-    public function __construct($fileLinkFormat = null, RequestStack $requestStack = null, $baseDir = null, $urlFormat = null)
+    public function __construct($fileLinkFormat = null, RouterInterface $router = null, $baseDir = null, $queryString = null)
```

Comment:
the typehint should be UrlGeneratorInterface, as yo only want to generate (Silex does not have a service implementing RouterInterface for instance, as it has 2 services for the matcher and the generator)